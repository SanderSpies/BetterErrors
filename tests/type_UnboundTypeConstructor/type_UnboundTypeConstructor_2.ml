type whereAm = {a: int}

type asd =
  | Hello of whereAmI
  | Goodbye
