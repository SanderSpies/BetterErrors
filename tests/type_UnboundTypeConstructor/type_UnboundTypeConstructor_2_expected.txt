tests/type_UnboundTypeConstructor/type_UnboundTypeConstructor_2.ml:4 13-21
  | Hello of whereAmI
             ^^^^^^^^ The type constructor whereAmI can't be found.
Hint: did you mean `whereAm`?
